src/note.rs

```diff
@@ -233,11 +276,25 @@ impl Note {
         self.value
     }
 
+    /// Returns the asset of this note.
+    pub fn asset(&self) -> AssetBase {
+        self.asset
+    }
+
     /// Returns the rseed value of this note.
     pub fn rseed(&self) -> &RandomSeed {
         &self.rseed
     }
 
+    /// Returns the rseed_split_note value of this note.
+    pub fn rseed_split_note(&self) -> CtOption<RandomSeed> {
```

Comment:
str4d ([Link](https://github.com/zcash/orchard/pull/471#discussion_r2543542313)): Does this need to be in the public API? If the APIs that take this value to recreate the note (currently `Note::set_rseed_split_note`, after my suggestion `Note::from_parts_internal`) are crate-private, then presumably this can also be crate-private:
```suggestion
    pub(crate) fn rseed_split_note(&self) -> CtOption<RandomSeed> {
```